### PR TITLE
Upgrade a couple dependencies

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Comment by benmccann** [comment]
@jroper @jamesward I couldn't figure out how to get the tests to pass with `webjars-locator` 0.24. We need to upgrade `sbt-web` in order to upgrade Play. Would you guys be able to figure out what's wrong?

>[info] java.io.FileNotFoundException: /tmp/sbt_73037fe/publish-webjar/target/web/web-modules/main/webjars/lib/web-module-a/js/a.js (Permission denied)
[info] 	at java.io.FileInputStream.open0(Native Method)
[info] 	at java.io.FileInputStream.open(FileInputStream.java:195)
[info] 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
[info] 	at sbt.Using$$anonfun$fileInputChannel$1.apply(Using.scala:83)
[info] 	at sbt.Using$$anonfun$fileInputChannel$1.apply(Using.scala:83)
[info] 	at sbt.Using$$anon$2.openImpl(Using.scala:72)
[info] 	at sbt.OpenFile$class.open(Using.scala:46)
[info] 	at sbt.Using$$anon$2.open(Using.scala:70)
[info] 	at sbt.Using$$anon$2.open(Using.scala:70)
[info] 	at sbt.Using.apply(Using.scala:24)
[info] 	at sbt.IO$.copyFile(IO.scala:582)
[info] 	at sbt.Sync$.copy(Sync.scala:52)
[info] 	at sbt.Sync$$anonfun$apply$1$$anonfun$apply$2.apply(Sync.scala:44)
[info] 	at sbt.Sync$$anonfun$apply$1$$anonfun$apply$2.apply(Sync.scala:44)
[info] 	at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:54)
[info] 	at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:53)
[info] 	at scala.collection.immutable.Stream.foreach(Stream.scala:547)
[info] 	at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:44)
[info] 	at sbt.Sync$$anonfun$apply$1.apply(Sync.scala:23)
[info] 	at com.typesafe.sbt.web.SbtWeb$.syncMappings(SbtWeb.scala:487)
[info] 	at com.typesafe.sbt.web.SbtWeb$$anonfun$34.apply(SbtWeb.scala:281)
[info] 	at com.typesafe.sbt.web.SbtWeb$$anonfun$34.apply(SbtWeb.scala:281)
[info] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
[info] 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
[info] 	at sbt.std.Transform$$anon$4.work(System.scala:64)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
[info] 	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
[info] 	at sbt.Execute.work(Execute.scala:244)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
[info] 	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
[info] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[info] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[info] 	at java.lang.Thread.run(Thread.java:745)

File: build.sbt [hunk 1]
```diff
@@ -4,9 +4,9 @@ description := "sbt web support"
 
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.3.9",
-  "org.webjars" % "webjars-locator" % "0.21",
+  "org.webjars" % "webjars-locator" % "0.23",
```
Comment (jamesward) [inline build.sbt]: Didn't you want `0.24` here?

Comment (benmccann) [inline build.sbt]: Yes, I do. But the tests fail with it and I'm not sure why. Would you be able to take a stab at getting it to 0.24? See my comment on the main page of the PR for the error message I'm getting

**Comment by jamesward** [comment]
I bumped locally to `0.24` and all tests are passing for me.

**Comment by benmccann** [comment]
@jamesward are you sure? did you run `sbt scripted`? because it's failing on Travis too. I updated this PR to 0.24 so that you can see the error

**Comment by jamesward** [comment]
Ah. I just ran `sbt test`.  The permissions on the file are wrong and I bet this is something @jroper screwed up with his latest `webjars-locator-core` changes.
```
~/Desktop/sbt-web/src/sbt-test/sbt-web/publish-webjar $ ls -al target/web/web-modules/main/webjars/lib/web-module-b/js/
total 0
drwxr-xr-x  2 j.ward  454177323  102 Jun 18 15:09 .
drwxr-xr-x  3 j.ward  454177323  102 Jun 18 15:09 ..
----------  1 j.ward  454177323    0 Jun 18 15:09 b.js
```

**Comment by jroper** [comment]
I think the problem is, if the jar file doesn't have any permissions defined (like in this case, since the webjar was created by sbt which doesn't maintain permissions), then the resulting file gets a mode of `0000`.

**Comment by jroper** [comment]
Fix here:

https://github.com/webjars/webjars-locator-core/pull/5

**Comment by jamesward** [comment]
`webjars-locator` `0.25` has been released and should fix this.

**Comment by benmccann** [comment]
I've updated to `0.25` and the build is passing so this can be merged now

**Comment by benmccann** [comment]
Thanks @huntc !  Would you also be able to cut a new release?  @jroper and I wanted to get this fix into Play

**Pull request merged**